src/devices/machine/wd2010.cpp

```diff
@@ -57,7 +57,7 @@ UNIMPLEMENTED FEATURES :
 // #define CYLINDER_HIGH_MASK 0x03
 
 // --------------------------------------------------------
-#define MAX_MFM_SECTORS 17      // STANDARD MFM SECTORS/TRACK
+#define MAX_MFM_SECTORS 32      // STANDARD MFM SECTORS/TRACK
```

Comment:
Is the comment correct?  “Standard” PC 3600 RPM drives had 17 sectors of 512 bytes per track, didn’t they?

Are the drives you need to support 32 sectors of 256 bytes each?

Reply:
Yes, HP used 32 sectors with 256 bytes each. The comment is indeed not correct anymore as this is not "Standard". However, as MAX_MFM_SECTORS is the maximum number of sector supported, i wonder whether the comment should be just removed. It should be changed to at least something along "Maximum number of sectors this driver supports", but that would be documenting the obvious.

Reply:
I’m fine with just removing the comment – the name is fairly obvious.